perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,40 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm, vm_group_name):
+  """Installs the Cassandra charm on the VM."""
+  vm.JujuDeploy('cs:trusty/cassandra', vm_group_name)
+
+  # The charm defaults to Cassandra 2.2.x, which has deprecated
+  # cassandra-cli. Specify the sources to downgrade to Cassandra 2.1.x
+  # to match the cassandra benchmark(s) expectations.
+  sources = ['deb http://www.apache.org/dist/cassandra/debian 21x main',
+             'ppa:openjdk-r/ppa',
+             'ppa:stub/cassandra']
+
+  keys = ["F758CE318D77295D",
```

Comment:
super nit: the style guide suggests sticking to either single quotes or double quotes and then not switching (except docstrings which are always double quotes) unless switching eliminates the need to escape quotes (e.g. even if ' is standard, you can use " ' " over  ' \' '). We've generally used ' - could you switch to using ' except when the string contains a ' ? [Here's](https://google.github.io/styleguide/pyguide.html?showone=Strings#Strings) the style guide section